alacritty_terminal/src/grid/mod.rs

```diff
@@ -279,125 +280,180 @@ impl<T: GridCell + PartialEq + Copy> Grid<T> {
         self.display_offset = self.display_offset.saturating_sub(*lines_added);
     }
 
-    fn grow_cols(
-        &mut self,
-        reflow: bool,
-        cols: index::Column,
-        cursor_pos: &mut Point,
-        template: &T,
-    ) {
+    // Grow number of columns in each row, reflowing if necessary
+    fn grow_cols(&mut self, reflow: bool, cols: Column, cursor_pos: &mut Point, template: &T) {
+        // Check if a row needs to be wrapped
```

Comment:
It does the checking itself I'd say and returns a `bool` whether the checking was successful and the row needs a wrap or if it doesn't. So I feel like this is accurate, though it is true that it's a function that is later executed, not immediately.

I'll see if I can put that into the doc without blowing it up.

Reply:
Just looking at this comment again I feel like I had a hard times yesterday, since it's just a function to be honest, and it follows other function comments, so things should be fine the way they are right now.